src/bridge/ERC20Outbox.sol

```diff
@@ -5,11 +5,41 @@
 pragma solidity ^0.8.4;
 
 import "./AbsOutbox.sol";
+import {IERC20Bridge} from "./IERC20Bridge.sol";
+import {DecimalsConverterHelper} from "../libraries/DecimalsConverterHelper.sol";
+import {AmountTooLarge, NativeTokenDecimalsTooLarge} from "../libraries/Error.sol";
 
 contract ERC20Outbox is AbsOutbox {
-    // it is assumed that arb-os never assigns this value to a valid leaf to be redeemed
+    /// @dev number of decimals used by native token
+    uint8 public nativeTokenDecimals;
```

Comment:
Looks like this can be stored in [ERC20Bridge packed with nativeToken](https://github.com/OffchainLabs/nitro-contracts/blob/686c997cb82b95a6b7d4008613d750572d320c91/src/bridge/ERC20Bridge.sol#L23) to save gas (1 cold sload vs 1 warm call) ; also does not need a initializer